osutil/mockable.go

```diff
@@ -25,12 +25,24 @@ import (
 	"syscall"
 )
 
-const (
-	// ProcSelfMountInfo is a path to the mountinfo table of the current process.
-	ProcSelfMountInfo = "/proc/self/mountinfo"
-)
+// MockMountInfo is meant for tests to mock the content of /proc/self/mountinfo.
+func MockMountInfo(content string) (restore func()) {
+	old := mockedMountInfoContent
+	// the boolean is necessary because many tests will want to mock mountinfo
```

Comment:
Or we could just use a pointer to a string. Not a biggie, this is fine.

Reply:
yes, that's more typical,  we also tend to use *bool for similar cases when it's a bool

Reply:
done, now use a string point